### PR TITLE
Cypress: new Redact and NoRedact tests

### PR DESCRIPTION
**Pull request by AdamKol-up9**

### CODE REVIEW

File: acceptanceTests/cypress/integration/tests/NoRedact.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+it('Loading Mizu', function () {
+    cy.visit(Cypress.env('testUrl'));
+});
+
+['redact', 'REDACT', '{ "User": "[REDACTED]" }', '{"User":"[REDACTED]"}'].map(doItFunc)
```
Comment (AdamKol-up9) [inline acceptanceTests/cypress/integration/tests/NoRedact.js:5]: [REDACTED]

File: acceptanceTests/cypress/integration/tests/NoRedact.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+function doItFunc(stringToCheck) {
```
Comment (AdamKol-up9) [inline acceptanceTests/cypress/integration/tests/NoRedact.js:7]: do what func

File: acceptanceTests/tap_test.go [hunk 3]
```diff
@@ -233,6 +232,30 @@ func TestTapMultipleNamespaces(t *testing.T) {
 		t.Errorf("failed to start tap pods on time, err: %v", err)
 		return
 	}
+	podsUrl := fmt.Sprintf("%v/status/tap", apiServerUrl)
```
Comment (AdamKol-up9) [inline acceptanceTests/tap_test.go:235]: delete

**Review by RoyUP9 — APPROVED** [review]

File: acceptanceTests/cypress/integration/tests/RedactTests.js [hunk 4]
```diff
@@ -0,0 +1,24 @@
+const inHeader = 'User-Header[REDACTED]';
+// const inBody = '{ "User": "[REDACTED]" }';
```
Comment (RoyUP9) [inline acceptanceTests/cypress/integration/tests/RedactTests.js:2]: comment

File: acceptanceTests/cypress/integration/tests/RedactTests.js [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+const inBpdu = Cypres.env('third') ? '{ "User": "[REDACTED]" }';:
```
Comment (RoyUP9) [inline acceptanceTests/cypress/integration/tests/RedactTests.js:3]: this will not work

**Review by lirazyehezkel — APPROVED** [review]

**Pull request merged**